backend/src/infrastructure/http/routes/public/index.ts

```diff
@@ -1,10 +1,20 @@
-import { authRouter } from './auth';
-import { getProductRoute } from './getProduct';
-import { getProductsRoute } from './getProducts';
+import { authLoginRoute } from './authLogin';
+import { authResetPasswordRoute } from './authResetPassword';
+import { authResetPasswordRequestRoute } from './authResetPasswordRequest';
+import { healthcheckRoute } from './healthcheck';
+import { productGetListRoute } from './productGetList';
+import { productGetOneRoute } from './productGetOne';
 import { getHonoApp } from '../../config/getHonoApp';
 
 const publicRouter = getHonoApp();
 
-publicRouter.route('/', getProductRoute).route('/', getProductsRoute).route('/', authRouter);
+publicRouter
+  .route('/', authLoginRoute)
```

Comment:
Est ce qu'on ferai pas un truc vue qu'on regroupe par un mot clé "auth", "product"...

```suggestion
  .route('/auth', authLoginRoute)
```

Comme ça toute nos routes sont préfixé par leur mot clé et c'est plus facile du coup de retrouver les endpoints par rapport à la route